### ISSUE TITLE
[Bug]: Streaming child nodes, parent is not catching up

### ISSUE THREAD

**Issue by Forza-tng** [netdata/netdata#18100]
### Bug description

The data from the streaming child node is not updated in realtime, and it does not catch up - the delay is increasing more and more over time.

### Expected behavior

Real-time update of child nodes charts on the parent.

### Steps to reproduce

Using Alpine Linux 3.20.1
1. Install two nodesm; one parent and one child
2. Set up streaming from the child to the parent
3. Start the parent
4. Start the child
5. Watch the charts from the parent webui.
...

### Installation method

from git

### System info

```shell
Linux 6.6.33-0-virt #1-Alpine SMP PREEMPT_DYNAMIC Thu, 13 Jun 2024 07:49:22 +0000 x86_64 Linux
/etc/alpine-release:3.20.1
/etc/os-release:NAME="Alpine Linux"
/etc/os-release:ID=alpine
/etc/os-release:VERSION_ID=3.20.1
/etc/os-release:PRETTY_NAME="Alpine Linux v3.20"
```

### Netdata build info

```shell
Packaging:
    Netdata Version ____________________________________________ : v1.46.0-103-nightly
    Installation Type __________________________________________ : custom
    Package Architecture _______________________________________ : unknown
    Package Distro _____________________________________________ : unknown
    Configure Options __________________________________________ : dummy-configure-command
Default Directories:
    User Configurations ________________________________________ : /opt/netdata/etc/netdata
    Stock Configurations _______________________________________ : /opt/netdata/usr/lib/netdata/conf.d
    Ephemeral Databases (metrics data, metadata) _______________ : /opt/netdata/var/cache/netdata
    Permanent Databases ________________________________________ : /opt/netdata/var/lib/netdata
    Plugins ____________________________________________________ : /opt/netdata/usr/libexec/netdata/plugins.d
    Static Web Files ___________________________________________ : /opt/netdata/usr/share/netdata/web
    Log Files __________________________________________________ : /opt/netdata/var/log/netdata
    Lock Files _________________________________________________ : /opt/netdata/var/lib/netdata/lock
    Home _______________________________________________________ : /opt/netdata/var/lib/netdata
Operating System:
    Kernel _____________________________________________________ : Linux
    Kernel Version _____________________________________________ : 6.6.33-0-virt
    Operating System ___________________________________________ : Alpine Linux
    Operating System ID ________________________________________ : alpine
    Operating System ID Like ___________________________________ : unknown
    Operating System Version ___________________________________ : unknown
    Operating System Version ID ________________________________ : none
    Detection __________________________________________________ : /etc/os-release
Hardware:
    CPU Cores __________________________________________________ : 4
    CPU Frequency ______________________________________________ : 2794000000
    RAM Bytes __________________________________________________ : 986451968
    Disk Capacity ______________________________________________ : 6442450944
    CPU Architecture ___________________________________________ : x86_64
    Virtualization Technology __________________________________ : unknown
    Virtualization Detection ___________________________________ : none
Container:
    Container __________________________________________________ : unknown
    Container Detection ________________________________________ : none
    Container Orchestrator _____________________________________ : none
    Container Operating System _________________________________ : none
    Container Operating System ID ______________________________ : none
    Container Operating System ID Like _________________________ : none
    Container Operating System Version _________________________ : none
    Container Operating System Version ID ______________________ : none
    Container Operating System Detection _______________________ : none
Features:
    Built For __________________________________________________ : Linux
    Netdata Cloud ______________________________________________ : YES
    Health (trigger alerts and send notifications) _____________ : YES
    Streaming (stream metrics to parent Netdata servers) _______ : YES
    Back-filling (of higher database tiers) ____________________ : YES
    Replication (fill the gaps of parent Netdata servers) ______ : YES
    Streaming and Replication Compression ______________________ : YES (zstd lz4 gzip brotli)
    Contexts (index all active and archived metrics) ___________ : YES
    Tiering (multiple dbs with different metrics resolution) ___ : YES (5)
    Machine Learning ___________________________________________ : YES
Database Engines:
    dbengine (compression) _____________________________________ : YES (zstd lz4)
    alloc ______________________________________________________ : YES
    ram ________________________________________________________ : YES
    none _______________________________________________________ : YES
Connectivity Capabilities:
    ACLK (Agent-Cloud Link: MQTT over WebSockets over TLS) _____ : YES
    static (Netdata internal web server) _______________________ : YES
    h2o (web server) ___________________________________________ : YES
    WebRTC (experimental) ______________________________________ : NO
    Native HTTPS (TLS Support) _________________________________ : YES
    TLS Host Verification ______________________________________ : YES
Libraries:
    LZ4 (extremely fast lossless compression algorithm) ________ : YES
    ZSTD (fast, lossless compression algorithm) ________________ : YES
    zlib (lossless data-compression library) ___________________ : YES
    Brotli (generic-purpose lossless compression algorithm) ____ : YES
    protobuf (platform-neutral data serialization protocol) ____ : YES (system)
    OpenSSL (cryptography) _____________________________________ : YES
    libdatachannel (stand-alone WebRTC data channels) __________ : NO
    JSON-C (lightweight JSON manipulation) _____________________ : YES
    libcap (Linux capabilities system operations) ______________ : NO
    libcrypto (cryptographic functions) ________________________ : YES
    libyaml (library for parsing and emitting YAML) ____________ : YES
Plugins:
    apps (monitor processes) ___________________________________ : YES
    cgroups (monitor containers and VMs) _______________________ : YES
    cgroup-network (associate interfaces to CGROUPS) ___________ : YES
    proc (monitor Linux systems) _______________________________ : YES
    tc (monitor Linux network QoS) _____________________________ : YES
    diskspace (monitor Linux mount points) _____________________ : YES
    freebsd (monitor FreeBSD systems) __________________________ : NO
    macos (monitor MacOS systems) ______________________________ : NO
    statsd (collect custom application metrics) ________________ : YES
    timex (check system clock synchronization) _________________ : YES
    idlejitter (check system latency and jitter) _______________ : YES
    bash (support shell data collection jobs - charts.d) _______ : YES
    debugfs (kernel debugging metrics) _________________________ : YES
    cups (monitor printers and print jobs) _____________________ : NO
    ebpf (monitor system calls) ________________________________ : YES
    freeipmi (monitor enterprise server H/W) ___________________ : YES
    nfacct (gather netfilter accounting) _______________________ : YES
    perf (collect kernel performance events) ___________________ : YES
    slabinfo (monitor kernel object caching) ___________________ : YES
    Xen ________________________________________________________ : YES
    Xen VBD Error Tracking _____________________________________ : NO
    Logs Management ____________________________________________ : YES
Exporters:
    AWS Kinesis ________________________________________________ : NO
    GCP PubSub _________________________________________________ : NO
    MongoDB ____________________________________________________ : NO
    Prometheus (OpenMetrics) Exporter __________________________ : YES
    Prometheus Remote Write ____________________________________ : NO
    Graphite ___________________________________________________ : YES
    Graphite HTTP / HTTPS ______________________________________ : YES
    JSON _______________________________________________________ : YES
    JSON HTTP / HTTPS __________________________________________ : YES
    OpenTSDB ___________________________________________________ : YES
    OpenTSDB HTTP / HTTPS ______________________________________ : YES
    All Metrics API ____________________________________________ : YES
    Shell (use metrics in shell scripts) _______________________ : YES
Debug/Developer Features:
    Trace All Netdata Allocations (with charts) ________________ : NO
    Developer Mode (more runtime checks, slower) _______________ : NO
```

### Additional info

I have two nodes on the same network. The parent ihas IP 10.12.9.4 and the child is a VM with IP 10.12.9.11. No firewall enabled between them.

I use this script to build netdata from git sources:
```
#!/bin/bash
CPUFLAGS="-march=x86-64"
export CFLAGS="-O1 -ggdb $CPUFLAGS"
export CFLAGS="-O1 -ggdb $CPUFLAGS"
export CXXFLAGS="-O1 -ggdb $CPUFLAGS"
export NETDATA_CMAKE_OPTIONS="-DCMAKE_BUILD_TYPE=Debug"
export TMPDIR="/var/tmp/"

ND=`mktemp -d -p $TMPDIR netdata-build.XXXX` || exit 1
echo  Working dir is: "$ND"
pushd "$ND" > /dev/null
git clone https://github.com/netdata/netdata.git --depth=100 --recursive "$ND"
./netdata-installer.sh --enable-lto --enable-ml --build-json-c --disable-telemetry --install-prefix /opt --dont-start-it
```

### netdata.conf
```

[directories]
    log = /var/log/netdata
    cache = /var/cache/netdata
    home = /opt/netdata

[web]
    default port = 19999
    bind to = *
    allow badges from = *
    allow streaming from = *
    enable gzip compression = no
    web server threads = 6
    web server max sockets = 1024

[logs]
    errors flood protection period = 120
    errors to trigger flood protection = 1000

[plugins]
    netdata monitoring extended = yes
```
### stream.conf
```
[stream]
    enabled = no

[91DCD32F-8BF7-4573-816B-56468CBEF079]
    enabled = yes
    allow from = *
    default postpone alarms on connect seconds = 120
```

### netdata.conf
```
[global]
    run as user = netdata
    hostname = dxsrv11
[db]
    mode = ram
    retention = 3600
[ml]
    enabled = no

[health]
    enabled = no

[web]
    mode = none

[directories]
    config = /opt/netdata/etc/netdata
    log = /var/log/netdata
    cache = /var/cache/netdata
    home = /opt/netdata

[logs]
    errors flood protection period = 1200
    errors to trigger flood protection = 200
```
### stream.conf
```
[stream]
    enabled = yes
    destination = 10.12.9.4
    api key = 91DCD32F-8BF7-4573-816B-56468CBEF079
    default port = 19999
    send charts matching = *
    # 10MiB buffer
    buffer size bytes = 10485760
```

## Logs

The logs do not show very much.

On the parent node, onlyt the following is logged once the child node connects:
```
time=2024-07-10T12:45:42.001+02:00 comm=netdata source=daemon level=info tid=21251 thread=RCVR[dxsrv11] node=dxsrv11 src_transport=http src_ip=10.12.9.11 src_port=52436 src_capabilities="VCAPS HLABELS CLAIM CLABELS LZ4 FUNCTIONS REPLICATION BINARY INTERPOLATED IEEE754 DYNCFG SLOTS ZSTD GZIP PROGRESS " msg="STREAM dxsrv11 [10.12.9.11]:52436: receive thread started"
time=2024-07-10T12:45:42.001+02:00 comm=netdata source=daemon level=info tid=21251 thread=RCVR[dxsrv11] node=dxsrv11 src_transport=http src_ip=10.12.9.11 src_port=52436 src_capabilities="VCAPS HLABELS CLAIM CLABELS LZ4 FUNCTIONS REPLICATION BINARY INTERPOLATED IEEE754 DYNCFG SLOTS ZSTD GZIP PROGRESS " msg="Host 'dxsrv11' (at registry as 'dxsrv11') with guid '8d4ea1a4-3ea9-11ef-9a24-def2eb0842f1' initialized, os 'linux', timezone 'CEST', program_name 'netdata', program_version 'v1.46.0-120-nightly', update every 1, memory mode dbengine, history entries 0, streaming disabled (to '' with api key ''), health enabled, cache_dir '(null)', alarms default handler '', alarms default recipient ''"
time=2024-07-10T12:45:42.024+02:00 comm=netdata source=daemon level=info tid=21251 thread=RCVR[dxsrv11] node=dxsrv11 src_transport=http src_ip=10.12.9.11 src_port=52436 src_capabilities="VCAPS HLABELS CLAIM CLABELS FUNCTIONS REPLICATION BINARY INTERPOLATED IEEE754 DYNCFG SLOTS ZSTD PROGRESS " msg="STREAM dxsrv11 [receive from [10.12.9.11]:52436]: established link with negotiated capabilities: VCAPS HLABELS CLAIM CLABELS FUNCTIONS REPLICATION BINARY INTERPOLATED IEEE754 DYNCFG SLOTS ZSTD PROGRESS "
time=2024-07-10T12:45:42.024+02:00 comm=netdata source=daemon level=info tid=21251 thread=RCVR[dxsrv11] msg_id=ed4cdb8f1beb4ad3b57cb3cae2d162fa node=dxsrv11 src_transport=http src_ip=10.12.9.11 src_port=52436 src_capabilities="VCAPS HLABELS CLAIM CLABELS FUNCTIONS REPLICATION BINARY INTERPOLATED IEEE754 DYNCFG SLOTS ZSTD PROGRESS " code=CONNECTED msg="STREAM_RECEIVER for 'dxsrv11': connected and ready to receive data "
```
The child log shows:
```
time=2024-07-10T12:45:41.220+02:00 comm=netdata source=daemon level=info errno="17, File exists" tid=34203  msg="Netdata agent version 'v1.46.0-120-nightly' is starting"
time=2024-07-10T12:45:41.221+02:00 comm=netdata source=daemon level=info tid=34203  msg="IEEE754: system is using IEEE754 DOUBLE PRECISION values"
time=2024-07-10T12:45:41.221+02:00 comm=netdata source=daemon level=info errno="2, No such file or directory" tid=34203  msg="TIMEZONE: using strftime(): 'CEST'"
time=2024-07-10T12:45:41.221+02:00 comm=netdata source=daemon level=info tid=34203  msg="TIMEZONE: fixed as 'CEST'"
time=2024-07-10T12:45:41.221+02:00 comm=netdata source=daemon level=info tid=34203  msg="NETDATA STARTUP: next: initialize signals"
time=2024-07-10T12:45:41.221+02:00 comm=netdata source=daemon level=info tid=34203  msg="NETDATA STARTUP: in       0 ms, initialize signals - next: initialize static threads"
time=2024-07-10T12:45:41.221+02:00 comm=netdata source=daemon level=info tid=34203  msg="NETDATA STARTUP: in       0 ms, initialize static threads - next: initialize web server"
time=2024-07-10T12:45:41.221+02:00 comm=netdata source=daemon level=info tid=34203  msg="NETDATA STARTUP: in       0 ms, initialize web server - next: initialize ML"
time=2024-07-10T12:45:41.222+02:00 comm=netdata source=daemon level=info tid=34203  msg="NETDATA STARTUP: in       0 ms, initialize ML - next: initialize h2o server"
time=2024-07-10T12:45:41.222+02:00 comm=netdata source=daemon level=info tid=34203  msg="NETDATA STARTUP: in       0 ms, initialize h2o server - next: set resource limits"
time=2024-07-10T12:45:41.222+02:00 comm=netdata source=daemon level=info tid=34203  msg="resources control: allowed file descriptors: soft = 1024, max = 4096"
time=2024-07-10T12:45:41.222+02:00 comm=netdata source=daemon level=info tid=34203  msg="NETDATA STARTUP: in       0 ms, set resource limits - next: become daemon"
time=2024-07-10T12:45:41.222+02:00 comm=netdata source=daemon level=info tid=34203  msg="Out-Of-Memory (OOM) score is already set to the wanted value 0"
time=2024-07-10T12:45:41.222+02:00 comm=netdata source=daemon level=error errno="38, Function not implemented" tid=34203  msg="Cannot adjust netdata scheduling policy to batch (3), with priority 0. Falling back to nice."
time=2024-07-10T12:45:41.222+02:00 comm=netdata source=daemon level=error errno="38, Function not implemented" tid=34203  msg="Cannot get my current process scheduling policy."
time=2024-07-10T12:45:41.222+02:00 comm=netdata source=daemon level=info tid=34203  msg="netdata started on pid 34203."
time=2024-07-10T12:45:41.222+02:00 comm=netdata source=daemon level=info tid=34203  msg="NETDATA STARTUP: in       0 ms, become daemon - next: initialize threads after fork"
time=2024-07-10T12:45:41.222+02:00 comm=netdata source=daemon level=info tid=34203  msg="NETDATA STARTUP: in       0 ms, initialize threads after fork - next: initialize registry"
time=2024-07-10T12:45:41.222+02:00 comm=netdata source=daemon level=info tid=34203  msg="Registry is disabled - use the central netdata"
time=2024-07-10T12:45:41.222+02:00 comm=netdata source=daemon level=info errno="17, File exists" tid=34203  msg="NETDATA STARTUP: in       0 ms, initialize registry - next: fork the spawn server"
time=2024-07-10T12:45:41.222+02:00 comm=netdata source=daemon level=info tid=34203  msg="Initializing spawn client."
time=2024-07-10T12:45:41.223+02:00 comm=netdata source=daemon level=info tid=34203  msg="NETDATA STARTUP: in       1 ms, fork the spawn server - next: collecting system info"
time=2024-07-10T12:45:41.291+02:00 comm=netdata source=daemon level=info tid=34203  msg="NETDATA STARTUP: in      67 ms, collecting system info - next: initialize RRD structures"
time=2024-07-10T12:45:41.291+02:00 comm=netdata source=daemon level=info tid=34203  msg="SQLite database /var/cache/netdata/netdata-meta.db initialization"
time=2024-07-10T12:45:41.298+02:00 comm=netdata source=daemon level=info errno="2, No such file or directory" tid=34203  msg="SQLite database initialization completed"
time=2024-07-10T12:45:41.298+02:00 comm=netdata source=daemon level=info tid=34203  msg="SQLite database /var/cache/netdata/context-meta.db initialization"
time=2024-07-10T12:45:41.303+02:00 comm=netdata source=daemon level=info tid=34203  msg="STREAM: added streaming destination No 1: '10.12.9.4' to host 'dxsrv11'"
time=2024-07-10T12:45:41.303+02:00 comm=netdata source=daemon level=info tid=34203  msg="Host 'dxsrv11' (at registry as 'dxsrv11') with guid '8d4ea1a4-3ea9-11ef-9a24-def2eb0842f1' initialized, os 'linux', timezone 'CEST', program_name 'netdata', program_version 'v1.46.0-120-nightly', update every 1, memory mode ram, history entries 4096, streaming enabled (to '10.12.9.4' with api key '91DCD32F-8BF7-4573-816B-56468CBEF079'), health enabled, cache_dir '/var/cache/netdata', alarms default handler '', alarms default recipient ''"
time=2024-07-10T12:45:41.305+02:00 comm=netdata source=daemon level=info tid=34203  msg="Creating archived hosts"
time=2024-07-10T12:45:41.305+02:00 comm=netdata source=daemon level=info tid=34203  msg="Created 0 archived hosts"
time=2024-07-10T12:45:41.305+02:00 comm=netdata source=daemon level=info tid=34203  msg="ACLK sync initialization completed"
time=2024-07-10T12:45:41.305+02:00 comm=netdata source=daemon level=info tid=34348 thread=ACLKSYNC msg="Starting ACLK synchronization thread"
time=2024-07-10T12:45:41.305+02:00 comm=netdata source=daemon level=info errno="2, No such file or directory" tid=34203  msg="NETDATA STARTUP: in      14 ms, initialize RRD structures - next: check for incomplete shutdown"
time=2024-07-10T12:45:41.305+02:00 comm=netdata source=daemon level=info errno="2, No such file or directory" tid=34203  msg="NETDATA STARTUP: in       0 ms, check for incomplete shutdown - next: collect claiming info"
time=2024-07-10T12:45:41.306+02:00 comm=netdata source=daemon level=info errno="2, No such file or directory" tid=34203  msg="Unable to load '/opt/netdata/var/lib/netdata/cloud.d/claimed_id', setting state to AGENT_UNCLAIMED"
time=2024-07-10T12:45:41.306+02:00 comm=netdata source=daemon level=info tid=34203  msg="NETDATA STARTUP: in       0 ms, collect claiming info - next: collect host labels"
time=2024-07-10T12:45:41.308+02:00 comm=netdata source=daemon level=info tid=34203  msg="NETDATA STARTUP: in       2 ms, collect host labels - next: start the static threads"
time=2024-07-10T12:45:41.309+02:00 comm=netdata source=daemon level=info errno="2, No such file or directory" tid=34381 thread=EXPORTING msg="CONFIG: cannot load user exporting config '/opt/netdata/etc/netdata/exporting.conf'. Will try the stock version."
time=2024-07-10T12:45:41.309+02:00 comm=netdata source=daemon level=info tid=34383 thread=ACLK_MAIN msg="Waiting for Cloud to be enabled"
time=2024-07-10T12:45:41.309+02:00 comm=netdata source=daemon level=info tid=34203  msg="NETDATA STARTUP: in       1 ms, start the static threads - next: initialize commands API"
time=2024-07-10T12:45:41.311+02:00 comm=netdata source=daemon level=info tid=34203  msg="Initializing command server."
time=2024-07-10T12:45:41.311+02:00 comm=netdata source=daemon level=info tid=34381 thread=EXPORTING msg="No connector instances to activate"
time=2024-07-10T12:45:41.311+02:00 comm=netdata source=daemon level=info tid=34381 thread=EXPORTING msg="EXPORTING: no exporting connectors configured"
time=2024-07-10T12:45:41.311+02:00 comm=netdata source=daemon level=info tid=34381 thread=EXPORTING msg="cleaning up..."
time=2024-07-10T12:45:41.312+02:00 comm=netdata source=daemon level=info tid=34203  msg="NETDATA STARTUP: in       2 ms, initialize commands API - next: ready"
time=2024-07-10T12:45:41.314+02:00 comm=netdata source=daemon level=info tid=34203  msg="NETDATA STARTUP: completed in 93 ms. Enjoy real-time performance monitoring!"
time=2024-07-10T12:45:41.379+02:00 comm=netdata source=daemon level=error tid=34388 thread=P[tc] msg="child pid 34399 exited with code 1."
time=2024-07-10T12:45:41.379+02:00 comm=netdata source=daemon level=info errno="10, No child process" tid=34203  msg="SIGNAL: waitid(34399): failed - it seems the child is already reaped"
time=2024-07-10T12:45:41.391+02:00 comm=netdata source=daemon level=info tid=34400 thread=PD[perf] module=plugins.d[perf.plugin] node=dxsrv11   src_transport=pluginsd request='DISABLE' msg="PLUGINSD: plugin called DISABLE. Disabling it."
time=2024-07-10T12:45:41.391+02:00 comm=netdata source=daemon level=error tid=34400 thread=PD[perf] module=plugins.d[perf.plugin] node=dxsrv11 src_transport=pluginsd msg="child pid 34434 exited with code 1."
time=2024-07-10T12:45:41.391+02:00 comm=netdata source=daemon level=error tid=34400 thread=PD[perf] module=plugins.d[perf.plugin] node=dxsrv11 src_transport=pluginsd msg="PLUGINSD: 'host:dxsrv11', '/opt/netdata/usr/libexec/netdata/plugins.d/perf.plugin' (pid 34434) exited with error code 1 and haven't collected any data. Disabling it."
time=2024-07-10T12:45:41.392+02:00 comm=netdata source=daemon level=info tid=34408 thread=PD[ioping] module=plugins.d[ioping.plugin] node=dxsrv11   src_transport=pluginsd request='DISABLE' msg="PLUGINSD: plugin called DISABLE. Disabling it."
time=2024-07-10T12:45:41.392+02:00 comm=netdata source=daemon level=error tid=34408 thread=PD[ioping] module=plugins.d[ioping.plugin] node=dxsrv11 src_transport=pluginsd msg="child pid 34432 exited with code 1."
time=2024-07-10T12:45:41.392+02:00 comm=netdata source=daemon level=error tid=34408 thread=PD[ioping] module=plugins.d[ioping.plugin] node=dxsrv11 src_transport=pluginsd msg="PLUGINSD: 'host:dxsrv11', '/opt/netdata/usr/libexec/netdata/plugins.d/ioping.plugin' (pid 34432) exited with error code 1 and haven't collected any data. Disabling it."
time=2024-07-10T12:45:41.393+02:00 comm=netdata source=daemon level=error tid=34407 thread=PD[python.d] module=plugins.d[python.d.plugin] node=dxsrv11   src_transport=pluginsd request="'ERROR' 'python' 'IS' 'NOT' 'AVAILABLE' 'IN' 'THIS' 'SYSTEM'" msg="PLUGINSD: parser_action('ERROR') failed on line 1: { 'ERROR' 'python' 'IS' 'NOT' 'AVAILABLE' 'IN' 'THIS' 'SYSTEM' } (quotes added to show parsing)"
time=2024-07-10T12:45:41.393+02:00 comm=netdata source=daemon level=error tid=34407 thread=PD[python.d] module=plugins.d[python.d.plugin] node=dxsrv11 src_transport=pluginsd msg="child pid 34436 exited with code 1."
time=2024-07-10T12:45:41.393+02:00 comm=netdata source=daemon level=error tid=34407 thread=PD[python.d] module=plugins.d[python.d.plugin] node=dxsrv11 src_transport=pluginsd msg="PLUGINSD: 'host:dxsrv11', '/opt/netdata/usr/libexec/netdata/plugins.d/python.d.plugin' (pid 34436) exited with error code 1 and haven't collected any data. Disabling it."
time=2024-07-10T12:45:41.499+02:00 comm=netdata source=daemon level=info tid=34406 thread=PD[charts.d] module=plugins.d[charts.d.plugin] node=dxsrv11   src_transport=pluginsd request='DISABLE' msg="PLUGINSD: plugin called DISABLE. Disabling it."
time=2024-07-10T12:45:41.500+02:00 comm=netdata source=daemon level=info tid=34406 thread=PD[charts.d] module=plugins.d[charts.d.plugin] node=dxsrv11 src_transport=pluginsd msg="PLUGINSD: 'host:dxsrv11', '/opt/netdata/usr/libexec/netdata/plugins.d/charts.d.plugin' (pid 34430) does not generate useful output but it reports success (exits with 0). Will not start it again - it is now disabled.."
time=2024-07-10T12:45:42.000+02:00 comm=netdata source=daemon level=notice tid=34377 thread=STATS_GLOBAL msg="STREAM dxsrv11 [send]: not ready - collected metrics are not sent to parent."
time=2024-07-10T12:45:42.000+02:00 comm=netdata source=daemon level=info tid=34584 thread=SNDR[dxsrv11] node=dxsrv11 src_capabilities="V1 V2 VN VCAPS HLABELS CLAIM CLABELS LZ4 FUNCTIONS REPLICATION BINARY INTERPOLATED IEEE754 DYNCFG SLOTS ZSTD GZIP BROTLI PROGRESS " dst_transport=http   msg="STREAM dxsrv11 [send]: thread created (task id 34584)"
time=2024-07-10T12:45:42.024+02:00 comm=netdata source=daemon level=info tid=34584 thread=SNDR[dxsrv11] msg_id=6e2e3839067648968b646045dbf28d66 node=dxsrv11 src_capabilities="VCAPS HLABELS CLAIM CLABELS FUNCTIONS REPLICATION BINARY INTERPOLATED IEEE754 DYNCFG SLOTS ZSTD PROGRESS " dst_transport=http dst_ip=10.12.9.4 dst_port=19999 msg="STREAM dxsrv11 [send to 10.12.9.4]: established link with negotiated capabilities: VCAPS HLABELS CLAIM CLABELS FUNCTIONS REPLICATION BINARY INTERPOLATED IEEE754 DYNCFG SLOTS ZSTD PROGRESS "
time=2024-07-10T12:45:42.311+02:00 comm=netdata source=daemon level=info tid=34394 thread="P[proc netdev]" msg="STREAM dxsrv11 [send]: sending metrics to parent..."
time=2024-07-10T12:45:42.314+02:00 comm=netdata source=daemon level=info tid=34390 thread=P[proc] module=proc.plugin[/proc/uptime] msg="Using now_boottime_usec() for uptime (dt is 5 ms)"
time=2024-07-10T12:45:53.336+02:00 comm=netdata source=daemon level=error tid=34402 thread=P[cgroupsdisc] msg="child pid 34599 exited with code 1."
time=2024-07-10T12:45:53.353+02:00 comm=netdata source=daemon level=error tid=34402 thread=P[cgroupsdisc] msg="child pid 34617 exited with code 1."
time=2024-07-10T12:45:53.360+02:00 comm=netdata source=daemon level=info tid=34203  msg="SIGNAL: reap_child(34629) exited with code: 0"
time=2024-07-10T12:45:53.371+02:00 comm=netdata source=daemon level=error tid=34402 thread=P[cgroupsdisc] msg="child pid 34633 exited with code 1."
time=2024-07-10T12:45:53.378+02:00 comm=netdata source=daemon level=info errno="10, No child process" tid=34203  msg="SIGNAL: waitid(34647): failed - it seems the child is already reaped"
time=2024-07-10T12:45:53.388+02:00 comm=netdata source=daemon level=error tid=34402 thread=P[cgroupsdisc] msg="child pid 34651 exited with code 1."
time=2024-07-10T12:45:53.394+02:00 comm=netdata source=daemon level=info errno="10, No child process" tid=34203  msg="SIGNAL: waitid(34663): failed - it seems the child is already reaped"
time=2024-07-10T12:45:53.405+02:00 comm=netdata source=daemon level=error tid=34402 thread=P[cgroupsdisc] msg="child pid 34667 exited with code 1."
time=2024-07-10T12:45:53.428+02:00 comm=netdata source=daemon level=error tid=34402 thread=P[cgroupsdisc] msg="child pid 34683 exited with code 1."
time=2024-07-10T12:45:53.435+02:00 comm=netdata source=daemon level=info errno="10, No child process" tid=34203  msg="SIGNAL: waitid(34707): failed - it seems the child is already reaped"
time=2024-07-10T12:45:53.459+02:00 comm=netdata source=daemon level=error tid=34402 thread=P[cgroupsdisc] msg="child pid 34711 exited with code 1."
time=2024-07-10T12:45:53.476+02:00 comm=netdata source=daemon level=error tid=34402 thread=P[cgroupsdisc] msg="child pid 34749 exited with code 1."
time=2024-07-10T12:46:43.829+02:00 comm=netdata source=daemon level=info tid=34387 thread=REPLAY[1] msg="REPLICATION SUMMARY: finished, executed 790 replication requests, 0 charts pending replication"
time=2024-07-10T12:47:41.947+02:00 comm=netdata source=daemon level=error errno="110, Operation timed out" tid=34393 thread=PD[nfacct] module=plugins.d[nfacct.plugin] node=dxsrv11   src_transport=pluginsd  msg="PARSER: timeout while waiting for data."
time=2024-07-10T12:47:41.948+02:00 comm=netdata source=daemon level=info tid=34393 thread=PD[nfacct] module=plugins.d[nfacct.plugin] node=dxsrv11 src_transport=pluginsd msg="PLUGINSD: 'host:dxsrv11', '/opt/netdata/usr/libexec/netdata/plugins.d/nfacct.plugin' (pid 34395) does not generate useful output but it reports success (exits with 0). Waiting a bit before starting it again.."
time=2024-07-10T12:49:52.604+02:00 comm=netdata source=daemon level=error errno="110, Operation timed out" tid=34393 thread=PD[nfacct] module=plugins.d[nfacct.plugin] node=dxsrv11   src_transport=pluginsd  msg="PARSER: timeout while waiting for data."
time=2024-07-10T12:49:52.604+02:00 comm=netdata source=daemon level=info tid=34393 thread=PD[nfacct] module=plugins.d[nfacct.plugin] node=dxsrv11 src_transport=pluginsd msg="PLUGINSD: 'host:dxsrv11', '/opt/netdata/usr/libexec/netdata/plugins.d/nfacct.plugin' (pid 34789) does not generate useful output but it reports success (exits with 0). Waiting a bit before starting it again.."
time=2024-07-10T12:49:52.604+02:00 comm=netdata source=daemon level=info errno="10, No child process" tid=34203  msg="SIGNAL: waitid(34789): failed - it seems the child is already reaped"
time=2024-07-10T12:52:03.258+02:00 comm=netdata source=daemon level=error errno="110, Operation timed out" tid=34393 thread=PD[nfacct] module=plugins.d[nfacct.plugin] node=dxsrv11   src_transport=pluginsd  msg="PARSER: timeout while waiting for data."
time=2024-07-10T12:52:03.258+02:00 comm=netdata source=daemon level=info tid=34393 thread=PD[nfacct] module=plugins.d[nfacct.plugin] node=dxsrv11 src_transport=pluginsd msg="PLUGINSD: 'host:dxsrv11', '/opt/netdata/usr/libexec/netdata/plugins.d/nfacct.plugin' (pid 34822) does not generate useful output but it reports success (exits with 0). Waiting a bit before starting it again.."
```

## Charts
This is how the charts behave for the child node. 
### Immediately after starting the child:
![image](https://github.com/netdata/netdata/assets/68693597/e4748467-4472-462d-a2bd-87262f4617a5)
### After a new minutes
![image](https://github.com/netdata/netdata/assets/68693597/32fd39e3-a235-4584-9d40-c9ab1cb08abb)
### After some more time
![image](https://github.com/netdata/netdata/assets/68693597/70798eec-9740-4511-b2b2-e3d3400a5abd)
### After some more time
![image](https://github.com/netdata/netdata/assets/68693597/159d12df-6ffd-4fa6-86ba-afd5380adaab)

### Parent node
There are no gaps in any charts on the parent node. It behaves normally with no apparent issues:
![image](https://github.com/netdata/netdata/assets/68693597/e2cae32c-523b-4ded-add8-1c6569b6afea)

**Comment by ilyam8** [comment]
Hi, @Forza-tng. Why do you use your script instead of the netdata installer? Can you try the same setup using static build for example?

**Comment by Forza-tng** [comment]
> Hi, @Forza-tng. Why do you use your script instead of the netdata installer? Can you try the same setup using static build for example?

How do you mean? The script is the same as described here: https://learn.netdata.cloud/docs/developer-and-contributor-corner/install-the-netdata-agent-from-a-git-checkout

**Comment by ilyam8** [comment]
You use [kickstart with --static-only](https://learn.netdata.cloud/docs/netdata-agent/installation/linux/#install-type).

**Comment by Forza-tng** [comment]
> You use [kickstart with --static-only](https://learn.netdata.cloud/docs/netdata-agent/installation/linux/#install-type).
> 
> > where would I download the ones you publish
> 
> GitHub. Check [v1.46.1 assets](https://github.com/netdata/netdata/releases/tag/v1.46.1).

Thank you.

I did `rm -rf /opt/netdata`  and then `./kickstart.sh --static-only --no-updates --dont-start-it --disable-telemetry`

Unfortunately, the problem is exactly the same. Is there a way to debug the streaming parts to see why it is falling behind with the updates? Perhaps change to another method of sending the data, etc?

![image](https://github.com/netdata/netdata/assets/68693597/2bb9235e-d46f-41a2-a80c-c0274aaa2d42)

Log on the parent after starting the child:
```
time=2024-07-10T13:50:06.001+02:00 comm=netdata source=daemon level=info tid=2298 thread=RCVR[dxsrv11] node=dxsrv11 src_transport=http src_ip=10.12.9.11 src_port=58106 src_capabilities="VCAPS HLABELS CLAIM CLABELS LZ4 FUNCTIONS REPLICATION BINARY INTERPOLATED IEEE754 DYN
CFG SLOTS ZSTD GZIP PROGRESS " msg="STREAM dxsrv11 [10.12.9.11]:58106: receive thread started"
time=2024-07-10T13:50:06.001+02:00 comm=netdata source=daemon level=info tid=2298 thread=RCVR[dxsrv11] node=dxsrv11 src_transport=http src_ip=10.12.9.11 src_port=58106 src_capabilities="VCAPS HLABELS CLAIM CLABELS LZ4 FUNCTIONS REPLICATION BINARY INTERPOLATED IEEE754 DYN
CFG SLOTS ZSTD GZIP PROGRESS " msg="Host 'dxsrv11' (at registry as 'dxsrv11') with guid '8c3f660a-3eb2-11ef-a617-def2eb0842f1' initialized, os 'linux', timezone 'CEST', program_name 'netdata', program_version 'v1.46.0-120-g0cce79dfd', update every 1, memory mode dbengine
, history entries 0, streaming disabled (to '' with api key ''), health enabled, cache_dir '(null)', alarms default handler '', alarms default recipient ''"
time=2024-07-10T13:50:06.001+02:00 comm=netdata source=daemon level=info tid=2298 thread=RCVR[dxsrv11] node=dxsrv11 src_transport=http src_ip=10.12.9.11 src_port=58106 src_capabilities="VCAPS HLABELS CLAIM CLABELS FUNCTIONS REPLICATION BINARY INTERPOLATED IEEE754 DYNCFG
SLOTS ZSTD PROGRESS " msg="STREAM dxsrv11 [receive from [10.12.9.11]:58106]: established link with negotiated capabilities: VCAPS HLABELS CLAIM CLABELS FUNCTIONS REPLICATION BINARY INTERPOLATED IEEE754 DYNCFG SLOTS ZSTD PROGRESS "
time=2024-07-10T13:50:06.001+02:00 comm=netdata source=daemon level=info tid=2298 thread=RCVR[dxsrv11] msg_id=ed4cdb8f1beb4ad3b57cb3cae2d162fa node=dxsrv11 src_transport=http src_ip=10.12.9.11 src_port=58106 src_capabilities="VCAPS HLABELS CLAIM CLABELS FUNCTIONS REPLICA
TION BINARY INTERPOLATED IEEE754 DYNCFG SLOTS ZSTD PROGRESS " code=CONNECTED msg="STREAM_RECEIVER for 'dxsrv11': connected and ready to receive data "
```

```
# /opt/netdata/usr/sbin/netdata -W buildinfo
Packaging:
    Netdata Version ____________________________________________ : v1.46.0-120-g0cce79dfd
    Installation Type __________________________________________ : kickstart-static
    Package Architecture _______________________________________ : x86_64
    Package Distro _____________________________________________ : unknown
    Configure Options __________________________________________ : dummy-configure-command
Default Directories:
    User Configurations ________________________________________ : /opt/netdata/etc/netdata
    Stock Configurations _______________________________________ : /opt/netdata/usr/lib/netdata/conf.d
    Ephemeral Databases (metrics data, metadata) _______________ : /opt/netdata/var/cache/netdata
    Permanent Databases ________________________________________ : /opt/netdata/var/lib/netdata
    Plugins ____________________________________________________ : /opt/netdata/usr/libexec/netdata/plugins.d
    Static Web Files ___________________________________________ : /opt/netdata/usr/share/netdata/web
    Log Files __________________________________________________ : /opt/netdata/var/log/netdata
    Lock Files _________________________________________________ : /opt/netdata/var/lib/netdata/lock
    Home _______________________________________________________ : /opt/netdata/var/lib/netdata
Operating System:
    Kernel _____________________________________________________ : Linux
    Kernel Version _____________________________________________ : 6.6.33-0-virt
    Operating System ___________________________________________ : Alpine Linux
    Operating System ID ________________________________________ : alpine
    Operating System ID Like ___________________________________ : unknown
    Operating System Version ___________________________________ : unknown
    Operating System Version ID ________________________________ : none
    Detection __________________________________________________ : /etc/os-release
Hardware:
    CPU Cores __________________________________________________ : 4
    CPU Frequency ______________________________________________ : 2794000000
    RAM Bytes __________________________________________________ : 986451968
    Disk Capacity ______________________________________________ : 6442450944
    CPU Architecture ___________________________________________ : x86_64
    Virtualization Technology __________________________________ : unknown
    Virtualization Detection ___________________________________ : none
Container:
    Container __________________________________________________ : unknown
    Container Detection ________________________________________ : none
    Container Orchestrator _____________________________________ : none
    Container Operating System _________________________________ : none
    Container Operating System ID ______________________________ : none
    Container Operating System ID Like _________________________ : none
    Container Operating System Version _________________________ : none
    Container Operating System Version ID ______________________ : none
    Container Operating System Detection _______________________ : none
Features:
    Built For __________________________________________________ : Linux
    Netdata Cloud ______________________________________________ : YES
    Health (trigger alerts and send notifications) _____________ : YES
    Streaming (stream metrics to parent Netdata servers) _______ : YES
    Back-filling (of higher database tiers) ____________________ : YES
    Replication (fill the gaps of parent Netdata servers) ______ : YES
    Streaming and Replication Compression ______________________ : YES (zstd lz4 gzip)
    Contexts (index all active and archived metrics) ___________ : YES
    Tiering (multiple dbs with different metrics resolution) ___ : YES (5)
    Machine Learning ___________________________________________ : YES
Database Engines:
    dbengine (compression) _____________________________________ : YES (zstd lz4)
    alloc ______________________________________________________ : YES
    ram ________________________________________________________ : YES
    none _______________________________________________________ : YES
Connectivity Capabilities:
    ACLK (Agent-Cloud Link: MQTT over WebSockets over TLS) _____ : YES
    static (Netdata internal web server) _______________________ : YES
    h2o (web server) ___________________________________________ : YES
    WebRTC (experimental) ______________________________________ : NO
    Native HTTPS (TLS Support) _________________________________ : YES
    TLS Host Verification ______________________________________ : YES
Libraries:
    LZ4 (extremely fast lossless compression algorithm) ________ : YES
    ZSTD (fast, lossless compression algorithm) ________________ : YES
    zlib (lossless data-compression library) ___________________ : YES
    Brotli (generic-purpose lossless compression algorithm) ____ : NO
    protobuf (platform-neutral data serialization protocol) ____ : YES (system)
    OpenSSL (cryptography) _____________________________________ : YES
    libdatachannel (stand-alone WebRTC data channels) __________ : NO
    JSON-C (lightweight JSON manipulation) _____________________ : YES
    libcap (Linux capabilities system operations) ______________ : NO
    libcrypto (cryptographic functions) ________________________ : YES
    libyaml (library for parsing and emitting YAML) ____________ : YES
Plugins:
    apps (monitor processes) ___________________________________ : YES
    cgroups (monitor containers and VMs) _______________________ : YES
    cgroup-network (associate interfaces to CGROUPS) ___________ : YES
    proc (monitor Linux systems) _______________________________ : YES
    tc (monitor Linux network QoS) _____________________________ : YES
    diskspace (monitor Linux mount points) _____________________ : YES
    freebsd (monitor FreeBSD systems) __________________________ : NO
    macos (monitor MacOS systems) ______________________________ : NO
    statsd (collect custom application metrics) ________________ : YES
    timex (check system clock synchronization) _________________ : YES
    idlejitter (check system latency and jitter) _______________ : YES
    bash (support shell data collection jobs - charts.d) _______ : YES
    debugfs (kernel debugging metrics) _________________________ : YES
    cups (monitor printers and print jobs) _____________________ : NO
    ebpf (monitor system calls) ________________________________ : YES
    freeipmi (monitor enterprise server H/W) ___________________ : NO
    nfacct (gather netfilter accounting) _______________________ : YES
    perf (collect kernel performance events) ___________________ : YES
    slabinfo (monitor kernel object caching) ___________________ : YES
    Xen ________________________________________________________ : NO
    Xen VBD Error Tracking _____________________________________ : NO
    Logs Management ____________________________________________ : NO
Exporters:
    AWS Kinesis ________________________________________________ : NO
    GCP PubSub _________________________________________________ : NO
    MongoDB ____________________________________________________ : NO
    Prometheus (OpenMetrics) Exporter __________________________ : YES
    Prometheus Remote Write ____________________________________ : YES
    Graphite ___________________________________________________ : YES
    Graphite HTTP / HTTPS ______________________________________ : YES
    JSON _______________________________________________________ : YES
    JSON HTTP / HTTPS __________________________________________ : YES
    OpenTSDB ___________________________________________________ : YES
    OpenTSDB HTTP / HTTPS ______________________________________ : YES
    All Metrics API ____________________________________________ : YES
    Shell (use metrics in shell scripts) _______________________ : YES
Debug/Developer Features:
    Trace All Netdata Allocations (with charts) ________________ : NO
    Developer Mode (more runtime checks, slower) _______________ : NO
```

**Comment by stelfrag** [comment]
@Forza-tng , thank you -- will try to reproduce. 

In the mean time, is it possible (as a test) to attempt to run the child with `mode = dbengine`

**Comment by Forza-tng** [comment]
> @Forza-tng , thank you -- will try to reproduce.
> 
> In the mean time, is it possible (as a test) to attempt to run the child with `mode = dbengine`

Sure. I'll check it when I'm back at work again tomorrow.

**Comment by ilyam8** [comment]
@stelfrag I can't reproduce the issue:

- created 2 Alpine 3.20.1 VMs.
- configure Parent/Child as described in the OP.

No issues.

**Comment by Forza-tng** [comment]
> @stelfrag I can't reproduce the issue:
> 
> * created 2 Alpine 3.20.1 VMs.
> * configure Parent/Child as described in the OP.
> 
> No issues.

Thank you for checking. The host here is a bare metal storage server with a bonded nic. But i have no drops logged on the hardware level - still, something in the network could be affecting things. I attempted to save that tcp payload, but it's binary. Is there a way to send plaintext data or decode the communication in other ways?

**Comment by stelfrag** [comment]
> Thank you for checking. The host here is a bare metal storage server with a bonded nic. But i have no drops logged on the hardware level - still, something in the network could be affecting things. I attempted to save that tcp payload, but it's binary. Is there a way to send plaintext data or decode the communication in other ways?

On the parent node netdata.conf, go to `[plugins]` section and set 

```
[plugins]                      
        netdata monitoring extended = yes
```

Restart the parent and in the dashboard under `Netdata Monitoring section`, there will be a

`workers streaming receive`

This will chart the streaming commands exchanged between parent - child

eg

![image](https://github.com/netdata/netdata/assets/52996999/52d4bcd3-a42a-42fd-964b-fa0a5f476ea6)

**Comment by Forza-tng** [comment]
This is how it looks:

![image](https://github.com/netdata/netdata/assets/68693597/4c02281c-f412-4147-b2bc-bb1c19cab61e)

**Comment by stelfrag** [comment]
> This is how it looks:
> 

Nothing suspicious here. Streaming seems to send about 330 charts/second

**Comment by Forza-tng** [comment]
> > This is how it looks:
> 
> Nothing suspicious here. Streaming seems to send about 330 charts/second

It seems that enabling dbengine has enabled the child node to catch up and is now 'up to date' in real-time. Before with only ram it never caught up and there are gaps with missing data.

<img width="1877" alt="image" src="https://github.com/netdata/netdata/assets/68693597/f7701ce2-dc78-4565-8bb8-7ba065f3f707">

**Comment by Forza-tng** [comment]
Here is another picture that shows the patchy retention of data before the dbengine (top chart) was enabled this morning.
![image](https://github.com/netdata/netdata/assets/68693597/dd1d0fa9-1cf4-41db-8829-434f9bd7a43b)

**Comment by Forza-tng** [comment]
Hi, I've left the system running continuously since previous message. The issue remains that children lag behind in the graphs. 

At the moment one child  is about 1.5 hours behind real-time. Another child is at 3 minutes behind.

![image](https://github.com/user-attachments/assets/eea6cc9c-ac31-4fdf-b0be-c94e41d4334c)

The parent node itself does not lag behind on any metrics as far as I can tell. 

These are the receive metrics. 
![image](https://github.com/user-attachments/assets/208b7138-0274-4a9a-902f-cd9334b5724c)

**Comment by stelfrag** [comment]
@Forza-tng thanks for this investigation -- from what I understand there is congestion during the normal streaming but not via the ssh tunnel.  I will try to reproduce by "causing" a congestion (investigating defaults etc)

**Comment by Forza-tng** [comment]
> @Forza-tng thanks for this investigation -- from what I understand there is congestion during the normal streaming but not via the ssh tunnel. I will try to reproduce by "causing" a congestion (investigating defaults etc)

Thanks. There is 'real' congesting between the two systems. Iperf3 with both TCP and UDP gives close to line speed of 1 gbit/s in both directions, so it's an odd occurrence. The Netdata parent is using a bonded network (LACP), perhaps that is affecting something, though I can't exactly say how it could be.

**Comment by stelfrag** [comment]
> > @Forza-tng thanks for this investigation -- from what I understand there is congestion during the normal streaming but not via the ssh tunnel. I will try to reproduce by "causing" a congestion (investigating defaults etc)
> 
> Thanks. There is no 'real' congestion between the two systems. Iperf3 with both TCP and UDP gives close to line speed of 1 gbit/s in both directions, so it's an odd occurrence. The Netdata parent is using a bonded network (LACP), perhaps that is affecting something, though I can't exactly say how it could be.

Can you check if by any change the clock on child nodes is drifting?

**Comment by Forza-tng** [comment]
> 
> Can you check if by any chance the clock on child nodes is drifting?

It does not appear to be. We use a central NTP server and Chrony to manage time.

Parent:
```
❯ chronyc tracking
Reference ID    : 8A6A0A19 (gm1.xxx.yy.com)
Stratum         : 3
Ref time (UTC)  : Wed Sep 04 10:43:25 2024
System time     : 0.000039772 seconds slow of NTP time
Last offset     : -0.000045419 seconds
RMS offset      : 0.000088692 seconds
Frequency       : 3.592 ppm slow
Residual freq   : -0.053 ppm
Skew            : 0.032 ppm
Root delay      : 0.002641915 seconds
Root dispersion : 0.003497926 seconds
Update interval : 1032.7 seconds
Leap status     : Normal
```

Child:
```
❯ chronyc tracking
Reference ID    : 8A6A0A19 (gm1.xxx.yy.com)
Stratum         : 3
Ref time (UTC)  : Wed Sep 04 10:43:50 2024
System time     : 0.000036766 seconds slow of NTP time
Last offset     : -0.000010837 seconds
RMS offset      : 0.000035774 seconds
Frequency       : 6.097 ppm fast
Residual freq   : -0.000 ppm
Skew            : 0.009 ppm
Root delay      : 0.002811465 seconds
Root dispersion : 0.002544495 seconds
Update interval : 1040.9 seconds
Leap status     : Normal
```

**Comment by Forza-tng** [comment]
This is a very odd problem for us. Perhaps one of the factors is that our Netdata child-nodes are running under Xen (XCP-ng). I get the same problem on `Ubuntu 22.04.5 6.8.0-45-generic` as with `Alpine Linux 3.20.3 6.6.52-0-virt`.

Sometimes one of the nodes start working in real-time, but if I restart Netdata or reboot it, it goes back to lagging. It seems that the TCP Window Full warning was not related after all. Even nodes without them can lag behind.

**Comment by Forza-tng** [comment]
I went and downloaded the [static version of 1.44.3](https://github.com/netdata/netdata/releases?page=2) and it started to look like it worked to start with.... until it didn't. Am I having a cursed system? 😨 

<img width="491" alt="image" src="https://github.com/user-attachments/assets/403e074a-a4dd-427d-881c-fb768fa37a39">

**Comment by ilyam8** [comment]
Closing this issue because we were unable to reproduce the problem, and it was reported on a very old version of Netdata.
